test/Android/androidtest.cmd

```diff
@@ -34,6 +35,14 @@ set _GlassDir=%_ProjectRoot%%_GlassPackageName%\
 if NOT exist "%_GlassDir%glass2.exe" echo Getting Glass from NuGet.& call "%_ProjectRoot%tools\NuGet\nuget.exe" install %_GlassPackageName% -Version %_GlassPackageVersion% -ExcludeVersion -OutputDirectory %_ProjectRoot%
 if NOT "%ERRORLEVEL%"=="0" echo ERROR: Failed to get Glass from NuGet.& exit /b -1
 
+:: Copy binaries to folder "Microsoft.VisualStudio.Glass" which required by glass2.exe in runtime for Visual Studio 2017
+if defined VS150COMNTOOLS (
+xcopy /Y /D "%VSINSTALLDIR%Common7\IDE\Remote Debugger\x86\Microsoft.VisualStudio.Debugger.Interop.15.0.dll" "%_GlassDir%"
+xcopy /Y /D "%VSINSTALLDIR%Common7\IDE\Remote Debugger\x86\Microsoft.VisualStudio.OLE.Interop.dll" "%_GlassDir%"
+xcopy /Y /D "%VSINSTALLDIR%Common7\IDE\PrivateAssemblies\Microsoft.VisualStudio.Shell.14.0.dll" "%_GlassDir%"
```

Comment:
Seems we should handle the exception in src/MIDebugEngine/AD7.Impl/AD7Engine.cs  instead, otherwise, glass will still throw the exception.